server/server.ts

```diff
@@ -278,6 +306,10 @@ app.delete('/deleteAccount', async (req, res) => {
   }
   if (postgres) {
     await postgres?.query('DELETE FROM room WHERE owner = $1', [decoded.uid]);
+    // set email to null to tell background process to clean it up.
+    await postgres?.query('UPDATE discord SET email = null WHERE email = $1', [
```

Comment:
couldn't we just delete it here?

Reply:
We could but then we could also add the role here and I don't know if this would defeat the purpose of having everything in one place in the syncSubs process?

Reply:
If we deleted the row here we would also have to remove the role from the Discord account here.

Reply:
Now that I think about it we only need to add the role on an account link. So seems like we should just do it here. Syncsubs could be used for cleanup later on